.prettierignore

```diff
@@ -4,10 +4,13 @@
 # Include specific folders
 !/code
 !/packages
-
 # But exclude folders anywhere
 node_modules
 dist
 build
 public
 .turbo
+
+# Exclude Handlebars.js (.hbs) files
+# TODO: Figure out how to format hanldebar templates
+**/*.hbs
```

Comment:
Prettier doesn't seem to be compatible with Handlebar templates (`*.hbs` files).
One solution that I'm thinking of is formatting the output after it has been generated, via a custom plop [action type](https://plopjs.com/documentation/#setactiontype).

```ts
// plopfile.ts

plop.setActionType("format", (answers, config, plop) => {
  // Use prettier.format() to format based on file extension
}

// ...

plop.setGenerator("constructBase", {
  actions: [
    // insert add action here,
   {
      type: "format",
      // ...
    }
  ]
})
```

Reply:
Alternatives to look into:

1. Use action's [transform](https://plopjs.com/documentation/#built-in-actions) method
```ts
plop.setGenerator("constructBase", {
  actions: [
    {
      type: "add",
      // ...
      transform: (template, data, config) => prettier.format(template, { filePath: config.path })
    }
  ]
})
```

2. Use plop plugin, [plop-action-eslint](https://github.com/natterstefan/plop-action-eslint).
```ts
plop.setGenerator("constructBase", {
  actions: [
    {
      type: "add",
      // ...
    },
    {
      type: "eslint",
      path: "src/app.tsx"
    }
  ]
})
```

Reply:
Using `transform` seemed like the cleanest implementation so I went with that: 

https://github.com/monstar-lab-oss/ml-frontend/pull/1249/files#diff-e0c0b2732525d78c9a3e09fc2671a8e56f0d6af9d07ec87c6afd502b924ae601R23-R25